### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 30

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#356] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152326082-1313dfb8-4e2f-4b10-8682-a9d4e9fa9062.png)

**Comment by ForzaNapolixsempre** [comment]
![30 logica](https://user-images.githubusercontent.com/122674373/212469167-1df00a36-46d9-48d1-afa1-4cd334c6cc9f.jpg)

**Comment by Alegau03** [comment]
> ![30 logica](https://user-images.githubusercontent.com/122674373/212469167-1df00a36-46d9-48d1-afa1-4cd334c6cc9f.jpg)

Perche se ti rimane Q metti un sat, è sat con q=1

**Comment by niccolo-fato** [comment]
> > ![30 logica](https://user-images.githubusercontent.com/122674373/212469167-1df00a36-46d9-48d1-afa1-4cd334c6cc9f.jpg)
> 
> Perche se ti rimane Q metti un sat, è sat con q=1

Lo elimina con {-Q} e esce una clausola vuota quindi dovrebbe essere UNSAT

**Comment by ATLANCTIX4** [comment]
come faccio a capire se alla fine è SAT o UNSAT?

Mi ritrovo alla fine con {q}, ma per determinare se la formula è SAT o UNSAT, è necessario:

Controllare il valore di q.
Se q è vero, la formula è SAT.
Se q è falso, la formula è UNSAT.
Se q non è stato assegnato un valore, la formula è in uno stato indeterminato.(?)

**Comment by francesco-alberghina** [comment]


**Reply by Exyss** [reply]
Non è possibile risolvere due clausole che hanno più di un letterale diverso. La formula è UNSAT ma questa refutazione è sbagliata.
